DQM/L1TMonitor/src/L1TStage2CaloLayer1.cc

```diff
@@ -706,10 +758,10 @@ void L1TStage2CaloLayer1::bookHistograms(DQMStore::IBooker& ibooker,
       bookHcalOccupancy("hcalOccFg2Discrepancy", "HCal Fine Grain 2 Discrepancy between uHTR and Layer1");
   eventMonitors.hcalOccFg3Discrepancy_ =
       bookHcalOccupancy("hcalOccFg3Discrepancy", "HCal Fine Grain 3 Discrepancy between uHTR and Layer1");
-  eventMonitors.hcalOccFg4Discrepancy_ =
-      bookHcalOccupancy("hcalOccFg4Discrepancy", "HCal Fine Grain 4 Discrepancy between uHTR and Layer1");
-  eventMonitors.hcalOccFg5Discrepancy_ =
-      bookHcalOccupancy("hcalOccFg5Discrepancy", "HCal Fine Grain 5 Discrepancy between uHTR and Layer1");
+  //eventMonitors.hcalOccFg4Discrepancy_ =
+  //    bookHcalOccupancy("hcalOccFg4Discrepancy", "HCal Fine Grain 4 Discrepancy between uHTR and Layer1");
+  //eventMonitors.hcalOccFg5Discrepancy_ =
+  //    bookHcalOccupancy("hcalOccFg5Discrepancy", "HCal Fine Grain 5 Discrepancy between uHTR and Layer1");
```

Comment:
These as well

Reply:
These are commented at the moment to mute monitoring of channels where HCAL is sending unphysical data to layer1, and will be uncommented once they fix it.